### PR TITLE
add a couple more team maxims

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

File: how_we_work/README.md [hunk 1]
```diff
@@ -28,3 +28,5 @@ See [meetings](policy/meetings.md) for more details.
 - We follow the [Hello, Production](https://blog.thepete.net/blog/2019/10/04/hello-production/) model.
 - Don't let the perfect be the enemy of the good.
 - We value both asynchronous and synchronous collaboration.
+- The team is distributed-first, though we take advantage of being in-person whenever possible.
+- If everything were easy / worked well, we wouldn't need to be here.
```
Comment (adborden) [inline how_we_work/README.md:32]: 🤔  this last one is unclear to me. This makes it seem like if we get TTS to a place where things are easy for programs, then we've done our job and can go home. That doesn't sound right to me, mind clarifying?

Comment (afeld) [reply how_we_work/README.md:32]: That’s kinda what I meant, yes! The Tech Portfolio exists because [gestures at everything] is hard 😄

Comment (adborden) [reply how_we_work/README.md:32]: I prefer the "force-multiplier" lens. If everything was easy, we'd still be here to amplify the things that are working.

I feel like the way it's written, I need to keep things hard so I can keep my job ;)

Comment (afeld) [reply how_we_work/README.md]: > I need to keep things hard so I can keep my job ;)

We should be so lucky as to be able to make everything easy! 😄 Anyway, rephrased to be more encouraging.

File: README.md [hunk 2]
```diff
@@ -31,20 +37,6 @@ TTS Tech Portfolio oversee/manage/coordinate everything technology-related that
...
 
-## Documentation
-
-See [the list of links](links.md) for more information about:
-
-- General Info
-- Money Info
-- Purchase card Documents
-- SaaS Management Documents
-- IaaS Management
```
Comment (afeld) [inline README.md:42]: This list was out of date.

File: README.md [hunk 3]
```diff
@@ -31,20 +37,6 @@ TTS Tech Portfolio oversee/manage/coordinate everything technology-related that
...
-
-## Team Process
-
-We are tracking the work for the Tech Portfolio team on our the public [18F GitHub project](https://github.com/orgs/18F/projects/11?fullscreen=true) board.
```
Comment (afeld) [inline README.md:46]: Linked to the workflow doc (which has more explanation) above.

File: README.md [hunk 4]
```diff
@@ -12,11 +12,17 @@ TTS Tech Portfolio oversee/manage/coordinate everything technology-related that
...
 - [Kanban](https://drive.google.com/file/d/0B0Qfvc1P_XBFVk5rM2tGcDBoUjQ/view)-ish and [Scrum](https://www.scrum.org/resources/what-is-scrum)-y
 - Sooner is better than later, later (or never) is better than sooner
+  - Try and deliver value quickly and learn from it, while seeing what we can defer to free up time for things that are more pressing.
 - Write things down
 - Discussions → action items
-- We use opinionated code/file formatters
-  - [`terraform fmt`](https://www.terraform.io/docs/commands/fmt.html)
-  - [Prettier](https://prettier.io/)
+- We practice [documentation-driven development](https://tom.preston-werner.com/2010/08/23/readme-driven-development.html)
+  - While documentation (e.g. [the Handbook](https://handbook.tts.gsa.gov/)) is generally the thing being updated (rather than a README), this should be done incrementally, and thus will be closer to README-Driven Development as it's defined in that post.
```
Comment (afeld) [inline README.md:19]: I feel like this explanation could be simpler; basically trying to say "practice README-Driven Development, but with documentation."

**Review by adborden — APPROVED** [review]

**Pull request merged**